specs/_features/eip7594/das-core.md

```diff
@@ -221,6 +224,48 @@ def get_data_column_sidecars(signed_block: SignedBeaconBlock,
     return sidecars
 ```
 
+#### `get_extended_sample_count`
+
+```python
+def get_extended_sample_count(allowed_failures: uint64) -> uint64:
+    assert 0 <= allowed_failures <= NUMBER_OF_COLUMNS // 2
```

Comment:
Nit: Never seen a function docstring below the asserts. Do we do this elsewhere in the specs?

Reply:
we did in phase0 `process_attestation`: https://github.com/ethereum/consensus-specs/blob/258c2c90205023a732c770c3496591456d4d6e7a/specs/phase0/beacon-chain.md?plain=1#L1824

Reply:
I've moved it, although a bit late. Anyway, if needed, here it is https://github.com/ethereum/consensus-specs/commit/195ec2163ab2d65996acbea147209b69dd84a610